schema/pet-insurance.json

```diff
@@ -346,6 +336,19 @@
                 "price"
             ]
         },
+        "VetPaymentTerm": {

```

Comment:
Can `PaymentTerm` be used instead? They look the same to me. Elimitate duplicate (identical) definitions, please, and add the descriptions to inputs/outputs instead.